### DISCUSSION TITLE
many java exception

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#813] [Q&A]
I'm trying to index a HD but after a few minutes the iped cancels.
Some information:
1 - This is a windows image dd.
2 - This HD is bitlocked, so I used ftkimager to mount the partitions, provided the bitlocker key and the HD was mounted on i:
3 - The line and command was:
java -Xms30G -Xmx30G -jar iped.jar --portable  -o c:\index_PGSP216  -profile forensic -d i:

I indexed anotger 3 HD in the same way without problems, only this one cancel.
here is the log.

[IPED-2021-10-30-18-18-25.log](https://github.com/sepinf-inc/IPED/files/7447651/IPED-2021-10-30-18-18-25.log)

any idea?

thank you
paulo

**Comment by wladimirleite** [comment]
I would try creating another image of the logical partition using FTK Imager and then process it, instead of processing a mounted volume.

From the log, I noticed some denied access issues, which are expected, as the mounted volume is accessed directly, so NTFS permissions may restrict the access.

Also I believe that FTK Imager mounting implementation is good enough to navigate in the volume, but not as "robust" as an actual volume. I observed issues when the mounted volume is accessed concurrently, which is the case when IPED process it.

**Reply by lfcnassif** [reply]
> From the log, I noticed some denied access issues, which are expected, as the mounted volume is accessed directly, so NTFS permissions may restrict the access.

I would just add that, even processing mounted folders as administrator, a lot of files in user's folders wouldn't be accessible, so I also recommend creating an image like @tc-wleite recommended.

**Comment by paulobreim** [comment]
Good idea.
I generated a DD and the iPed ended normally (although I have many java exceptions when reading files), but I don't know why I can't open the files for some permission issue, even though I'm admin. It is curious because if iped read and indexed the dd,there are no reason why iped search cannot read the files, windows can´t read too.
I will copy this dd to a linux and mount the image to see what happens.

**Comment by paulobreim** [comment]
After indexing all hd, any file i tried to view i got the message that ipedsearch could not open the file.
Checking manually I verified that the files really are not on the disk but in the cloud.
So what happens is that windows keeps file information like name, size, type etc, but the file itself is in the cloud so its content has not been indexed.
Reading some articles, apparently oxygen forensic is capable of detecting this type of situation, it discovers some credentials and saw it look for the file in the cloud.
This situation is new to me and complicates a forensic analysis a lot.

**Reply by lfcnassif** [reply]
This is related: https://github.com/sepinf-inc/IPED/issues/545

**Reply by paulobreim** [reply]
thank you. I will see
